res/values-es/strings.xml

```diff
@@ -76,74 +76,74 @@
     <string name="SelectAppToSend">Selecciona una aplicación para enviar</string>
     <string name="SettingsButtonContentDescription">Opciones</string>
     <string name="Share">Compartir</string>
-    <string name="SharePage">Compartir Página</string>
-    <string name="ShareSearch">Compartir Búsqueda</string>
-    <string name="ShareStory">Compartir Historia</string>
-    <string name="StoryOptionsTitle">Opciones de Historia</string>
-    <string name="SuggestSource">Sugerir una Fuente</string>
+    <string name="SharePage">Compartir página</string>
+    <string name="ShareSearch">Compartir búsqueda</string>
+    <string name="ShareStory">Compartir historia</string>
+    <string name="StoryOptionsTitle">Opciones de historia</string>
+    <string name="SuggestSource">Sugerir una fuente</string>
     <string name="SuggestSourceToast">Para implementar</string>
-    <string name="SuggestedSources">Por favor incluye estas fuentes:</string>
+    <string name="SuggestedSources">Por favor, incluye estas fuentes:</string>
     <string name="version">Versión</string>
     <string name="validity_period">Validez:</string>
     <string name="stories">Historias</string>
-    <string name="ssl_certificate_is_valid">Este certificado es valido</string>
-    <string name="ssl_certificate">Certificado de Seguridad</string>
+    <string name="ssl_certificate_is_valid">Este certificado es válido</string>
+    <string name="ssl_certificate">Certificado de seguridad</string>
     <string name="sha256_fingerprint">Huella SHA-256</string>
     <string name="sha1_fingerprint">Huella SHA-1</string>
     <string name="settings">Opciones</string>
-    <string name="serial_number">Número de Serie:</string>
-    <string name="select_application">Seleciona Aplicación</string>
+    <string name="serial_number">Número de serie:</string>
+    <string name="select_application">Selecionar aplicación</string>
     <string name="search_hint">Buscar&#8230;</string>
-    <string name="remove">Remover</string>
+    <string name="remove">Quitar</string>
     <string name="recents">Recientes</string>
     <string name="recent_stories_narrow">@string/stories</string>
-    <string name="recent_stories">Historias Recientes</string>
-    <string name="recent_searches_narrow">Busquedas</string>
-    <string name="recent_searches">Busquedas Recientes</string>
-    <string name="rate">Calificar Aplicación</string>
+    <string name="recent_stories">Historias recientes</string>
+    <string name="recent_searches_narrow">Búsquedas</string>
+    <string name="recent_searches">Búsquedas recientes</string>
+    <string name="rate">Calificar aplicación</string>
     <string name="org_unit">Unidad organizacional:</string>
     <string name="org_name">Organización:</string>
     <string name="options">Opciones:</string>
-    <string name="leave_feedback">Deja Retroalimentación</string>
+    <string name="leave_feedback">Dejar retroalimentación</string>
```

Comment:
What's this for?

* leave a comment or suggestion -> `Enviar comentario` or `Enviar sugerencia`
* stop sending feedback -> `Dejar de enviar información de uso`